Sources/Java2SwiftLib/StringExtras.swift

```diff
@@ -39,4 +39,18 @@ extension String {
   public func replacing(_ character: Character, with replacement: Character) -> String {
     return replacingOccurrences(of: String(character), with: String(replacement))
   }
+
+  public func optionalWrappedType() -> String? {
+    if self == "JavaOptionalInt!" {
+      return "Int32?"
+    } else if self == "JavaOptionalDouble!" {
+      return "Double?"
+    } else if self == "JavaOptionalLong!" {
+      return "Int64?"
```

Comment:
nitpick but maybe order as Int32, Int64 and then Double, grouping them closer together a bit